packages/microcosm/test/unit/observable.test.js

```diff
@@ -14,10 +14,28 @@ describe('Observable', function() {
           expect(reason).toEqual('Terrible things')
         }
       })
+
+      scheduler().flush()
+    })
+
+    it('cleans ups any enqueued tasks on unsubscribe', () => {
+      expect.assertions(1)
+
+      let observable = new Observable(observer => {
+        throw 'Terrible things'
+      })
+
+      observable.subscribe({
+        error(reason) {
+          expect(reason).toEqual('Terrible things')
+        }
+      })
+
+      scheduler().flush()
```

Comment:
Flush is a nice utility for forcing all updates to send out. You could also use async/await like `await scheduler`, but this has the nice benefit of test execution time.